scripts/filter-known-issues.py

```diff
@@ -59,13 +59,13 @@ def config_import_file(filename):
             # finditer() finds all the blocks and selects
             # the bits of mmapped-file that comprises
             # each--we compile it into a regex and append.
-            for m in re.finditer(config_regex, mm, re.MULTILINE):
+            for m in re.finditer(b"config_regex", mm, re.MULTILINE):
```

Comment:
@punitvara : there should be `b` before each opening doublequote, i.e.:
~~~
config_regex = \
    b"(?P<comment>(^\s*#.*\n)+)" \
    b"(?P<regex>(^[^#].*\n)+)"
~~~